components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -176,10 +173,13 @@ def custom_index(request, conn=None, **kwargs):
 ##############################################################################
 # views
 
+# from omeroweb.webgateway import LoginView
 
-def login(request):
+
+class WebclientLoginView(webgateway_views.LoginView):
```

Comment:
Added some tests https://github.com/openmicroscopy/openmicroscopy/pull/4708/commits/9843b25e36cb63ac2435ecb96d654d5be09870f0 and https://github.com/openmicroscopy/openmicroscopy/pull/4708/commits/e09e67cafae85dd084eccecc7494f4c4b0a5ce5a